Ultron/ultron.js

```diff
@@ -286,6 +287,10 @@
       },
     });
 
+		ref.addModule('edith', {
+      url_base: 'https://edith.apps.dev.rankmycluster.com',
```

Comment:
Como ficaria a mudança dessa var para a URL correta de prod?

Reply:
quando for pra branch master